README.md

```diff
@@ -21,7 +21,7 @@ This cookbook provides resources for the installation of Tripwire Enterprise Axo
 | `eg_driver_installer` | Defines the path to the installer package for the axon EG driver installer (linux/debian only) | String |
 | `eg_service_installer` | Defines the path to the installer package for the axon EG service installer (linux/debian only) | String |
 | `install_directory` | Install path for the Axon agent (current not in use) | String |
-| `eg_install` | Install the EG driver and service (linux/debian only) | Boolean |
+| `eg_install` | Install the EG driver and service (Future feature for Windows) | Boolean |
```

Comment:
It would probably be better to leave this as-is, rather than commenting on future features in the cookbook docs.

Reply:
Commented on @pfaffle, Java agents have a flag to not install the EG service, Windows axon agents might go that route and so eg_install would be a nice to have. It might also be useful if we have non-Tripwire Enterprise Axon agents that have no EG dependency (TLC?) It might be better to just pull the attribute till we are ready...